test/tstDetailsMortonCodes.cpp

```diff
@@ -43,6 +46,13 @@ BOOST_AUTO_TEST_CASE(expand_bits)
   BOOST_TEST(
       expandBitsBy<6>(y) ==
       0b000000100000010000001000000000000000000000000000000000000000001llu);
+  BOOST_TEST(expandBitsBy<7>(y) ==
```

Comment:
Please disable the format and align so that we can see better how the bit pattern changes from one line to the next

Reply:
Maybe anchor on the right most